### ISSUE TITLE
Log level none does not disable logging

### ISSUE THREAD

**Issue by dincho** [aeternity/aeternity#3568]
After deleting all contents of log/ directory and starting the node:

```
aeternity@ip-10-0-4-238:~/node$ grep level aeternity.yaml 
    level: none
aeternity@ip-10-0-4-238:~/node$ ls -la log/
total 104
drwxr-xr-x  2 aeternity aeternity  4096 Mar 29 18:12 .
drwxr-x--- 10 aeternity aeternity  4096 Mar 29 18:11 ..
-rw-rw-r--  1 aeternity aeternity     0 Mar 29 18:12 aestratum.log
-rw-rw-r--  1 aeternity aeternity  3683 Mar 29 18:12 aeternity.log
-rw-rw-r--  1 aeternity aeternity 71548 Mar 29 18:12 aeternity_metrics.log
-rw-rw-r--  1 aeternity aeternity     0 Mar 29 18:12 aeternity_mining.log
-rw-rw-r--  1 aeternity aeternity     0 Mar 29 18:12 aeternity_pow_cuckoo.log
-rw-rw-r--  1 aeternity aeternity     0 Mar 29 18:12 aeternity_sync.log
-rw-rw-r--  1 aeternity aeternity     0 Mar 29 18:12 crash.log
-rw-rw-r--  1 aeternity aeternity 15745 Mar 29 18:12 erlang.log.1
-rw-rw-r--  1 aeternity aeternity   304 Mar 29 18:12 run_erl.log
```

- I expect to see 0 bytes of logs. Although some boot logs in run_erl.log and aeternity.log are acceptable.
- Why the .1 suffix of erlang.log ?
- sync logs are kept logged to erlang.log:
```
aeternity@ip-10-0-4-238:~/node$ tail -f log/erlang.log.1 
18:15:19.599 [info] Synced blocks 128945 - 128947
18:15:19.786 [info] Peer "pp_g7KpMnE...3Ft7EyX" - TCP probe successful through process <0.4563.0>
18:15:20.534 [info] Connection accept failed - einval was from <<"3.64.216.76">> 
18:15:20.813 [info] Peer connections outbound: 4/10, inbound: 3/100, verified 8, unverified 78, standby 3
18:15:20.942 [info] Connection handshake failed - timeout was from <<"13.53.114.199">>
18:15:20.942 [info] Peer "pp_2gPZjuP...7FRvHTk" - TCP probe successful through process <0.4573.0>
18:15:21.469 [info] Connection accept failed - timeout was from <<"3.8.49.110">> 
18:15:21.931 [info] Peer "pp_ouZKBtB...45g71BD" - TCP probe successful through process <0.4621.0>
18:15:21.945 [info] Peer "pp_2gPZjuP...7FRvHTk" - TCP probe successful through process <0.4619.0>
```

**Comment by cna0** [comment]
@dincho `level: <log_level>` is only affecting `aeternity.log`, but in this case the log level is not working properly.

`erlang.log` is the console log which is redirected to a file. No configuration is affecting that by design.

I think `level: <log_level>` should set a log level for all logging backends. What do you think?

**Comment by richcarl** [comment]
Closing this since it was mostly fixed by #3691, apart from some remaining problems discussed in  #3863.

**Comment by richcarl** [comment]
> `erlang.log` is the console log which is redirected to a file. No configuration is affecting that by design.

Note for future reference: `erlang.log{.1, ..}` files are always created by the `run_erl` shim when running in the background (using the 'start' or 'daemon' flags), in order to capture stdout for debugging purposes. Normal systems are expected to use proper logging, not stdout, but sometimes you'll find critical info printed to stdout anyway, so the log file is good to have. To avoid this file creation, you have to run using the 'foreground' or 'console' flags which do not use `run_erl`.
